contracts/GovHub.template

```diff
@@ -66,7 +66,7 @@ contract GovHub is System, IParamSubscriber {
     uint256 totalVotes;
     bool canceled;
     bool executed;
-    mapping(address => Receipt) receipts;
+    mapping(address => Receipt) receipts; //@openissue
```

Comment:
inner mapping provides good encapsulation but it also prevents us from loading the struct into memory which could provide way better gas efficiency.

an alternative approach would be to store receipts in a separate mapping:
--- mapping(proposalId => mapping(address => Receipt))
making the struct (less the mapping) loadable to memory

Reply:
@jackcrypto9527 if that works for you I will do it in a separate PR so we can take the time and carefully test this change